src/shared/components/text-area/TextArea.tsx

```diff
@@ -0,0 +1,32 @@
+import { ChangeEvent, ComponentProps, forwardRef } from 'react'
+
+import { clsx } from 'clsx'
+
+import s from './TextArea.module.scss'
+type Props = {
+  errorMessage?: string
+  label?: string
+} & ComponentProps<'textarea'>
+export const Textarea = forwardRef<HTMLTextAreaElement, Props>(
+  ({ className, errorMessage, id, label, onChange, ...rest }, ref) => {
+    const showError = !!errorMessage && errorMessage.length > 0
```

Comment:
If errorMessage type is String | undefined and
 !!errorMessage === false 
 errorMessage.length will not be greater than zero
 This condition makes sense only if you are waiting Array or Object too